lib/util/tXml.js

```diff
@@ -0,0 +1,707 @@
+goog.provide('shaka.util.TXml');
+
+goog.require('shaka.util.StringUtils');
+goog.require('shaka.log');
+
+/**
+ * This code is a modified version of the tXml library.
+ *
+ * @author: Tobias Nickel
+ * created: 06.04.2015
+ * https://github.com/TobiasNickel/tXml
```

Comment:
I... think this means we need to modify our LICENSE file again?

I wish Joey was back, he knows this license stuff better than I do...

Reply:
Thanks for the review @theodab.
The license of the tXml library is MIT https://github.com/TobiasNickel/tXml/blob/master/LICENSE
I think that means that we just need to add the author to our author file? (I must admit I don't know either :) )

Reply:
Great I will do that now